network/p2p/network.go

```diff
@@ -333,7 +330,7 @@ func (n *Network) Receive(nodeID flow.Identifier, msg *message.Message) error {
 
 func (n *Network) processNetworkMessage(senderID flow.Identifier, message *message.Message) error {
 	// checks the cache for deduplication and adds the message if not already present
-	if n.rcache.add(message.EventID, network.Channel(message.ChannelID)) {
+	if !n.rcache.Add(message.EventID) {
```

Comment:
To be consistent with other `Add` interfaces in the codebase, the `Add` returns true for unseen messages.